src/test/integration/utils.py

```diff
@@ -58,3 +59,14 @@ def get_jobs_by_osu_id(osu_id):
 def get_image_by_osu_id(osu_id, params=None):
     global api_url, headers
     return requests.get(api_url + osu_id + '/image', headers=headers, params=params)
```

Comment:
Not your fault but there's a pep 8 error here: `utils.py:61:80: E501 line too long (84 > 79 characters)`